src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,60 @@
+$cell-side-size: 100px;
+$cell-gap-size: 1px;
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: 706px;
+  height: fit-content;
+  padding: 10px;
+  box-sizing: content-box;
+  gap: $cell-gap-size;
+
+  &__day {
+    background-color: #eee;
+    width: $cell-side-size;
+    height: $cell-side-size;
+    border: 1px solid #000;
+    box-sizing: border-box;
+    align-content: center;
```

Comment:
The `align-content` property is used for flex containers with multiple lines. Since `.calendar__day` is not a flex container, this property will not have any effect. Consider using `align-items` or `justify-content` if you want to align content within flex items.